src/resources/experimental/histograms-resource.js

```diff
@@ -1,12 +1,13 @@
 export default function HistogramsResource({apiHandler}) {
     return {
-        getTransactionHistogramReport({periodStart = '', periodEnd = '', aggregationField = '', aggregationPeriod = '', metric = '', criteria = null} = {}) {
+        getTransactionHistogramReport({periodStart = '', periodEnd = '', aggregationField = '', aggregationPeriod = '', metric = '', filter = '', criteria = null} = {}) {
```

Comment:
```suggestion
        getTransactionHistogramReport({periodStart = '', periodEnd = '', aggregationField = '', aggregationPeriod = '', metric = '', filter = null, criteria = null} = {}) {
```

Reply:
I wonder why this method has **criteria** at all?

Reply:
The report can parse criteria, but you're right, it shouldn't, so maybe you should remove that @dariashka (the `criteria`)